tests/python/pytorch/nn/test_nn.py

```diff
@@ -1393,10 +1393,7 @@ def test_cf_conv(g, idtype, out_dim):
 
 
 def myagg(alist, dsttype):
-    rst = alist[0]
-    for i in range(1, len(alist)):
-        rst = rst + (i + 1) * alist[i]
-    return rst
+    return alist[0] + np.dot([i + 1 for range(1, len(alist))],alist[1:])
```

Comment:
Why do we need to handle [0] separatedly? What about?
```suggestion
    return np.dot(list(range(1, len(alist) + 1), alist)
```